totalRP3/modules/register/main/register_tooltip.lua

```diff
@@ -497,12 +497,12 @@ local function writeTooltipForCharacter(targetID, _, targetType)
 
 	if showFullTitle() then
 		local fullTitle = "";
-		if info.characteristics and info.characteristics.FT and info.characteristics.FT:len() > 0 then
+		if info.characteristics and info.characteristics.FT and info.characteristics.FT ~= "" then
 			fullTitle = info.characteristics.FT;
 		elseif UnitPVPName(targetType) ~= targetName then
```

Comment:
Would changing this to 
```lua
elseif UnitPVPName(targetType)  and UnitPVPName(targetType) ~= targetName then
```
not be more consistant?

Reply:
It'd _work_ but it's an extra call to the same C function, and the end result is the same.

If we wanted to go down the rabbit hole of what's probably best, we'd not default the title to an empty string and instead _expect_ a nil value if the full title weren't available, noting that it can't ever be an empty string in either branch anyway (API bugs notwithstanding):

```lua
local fullTitle;

if info.characteristics and info.characteristics.FT and info.characteristics.FT ~= "" then
    fullTitle = info.characteristics.FT;
elseif UnitPVPName(targetType) ~= targetName then
    fullTitle = UnitPVPName(targetType);
end

if fullTitle then
```

Alas given the read-only nature of CurseForge in the next couple of weeks I'd rather not take the chance on a risky fix such as the above in the not-entirely-improbable case that UnitPVPName would return an empty string in some circumstance; especially as this isn't a regression in the most recent release but rather a previously unseen (and not likely easily reproduced) issue.

Reply:
counterpoint: Empty strings are not causing the error, nil values in `fullTitle` are. which admittedly, line 505 fixes.

I was mostly suggesting for the sake of consistency between the branches of the if. because at the moment we've got three outcome states where IMO we should be trying to have only two.
Either `""` and `"string"` or `nil` and `"string"`